sunpy/net/dataretriever/client.py

```diff
@@ -116,13 +117,11 @@ class GenericClient(BaseClient):
     is mainly designed for downloading data from FTP and HTTP type data
     sources, although should in theory be general enough to get data from any
     web service.
-
```

Comment:
Most of this whitespace changes needs to be undone.

Reply:
Oh I didn't noticed; probably I used reformatter.
I'll restore the previous.

Reply:
What reformatter?